tests/core/management/commands/test_find_missing_versions.py

```diff
@@ -47,6 +47,8 @@ def test_find_missing_versions_failure(load_test_data: None) -> None:
     """
     # Create version inconsistency
     page_translation = Page.objects.get(id=1).get_translation("de")
+    page_translation.slug = "willkommen"
```

Comment:
A comment would be nice. Thrown in a cheeky one for good measure
```suggestion
    # Prevent flaky test fails from the slug being different than expected
    # Yes this is a bad way to do this, feel free to rewrite our entire test suite yourself
    page_translation.slug = "willkommen"
```